mzLib/UsefulProteomicsDatabases/PtmListLoader.cs

```diff
@@ -213,6 +213,25 @@ public static IEnumerable<ModificationWithLocation> ReadMods(string ptmListLocat
             }
         }
 
+        public static IEnumerable<ModificationWithMass> ReadModsWithMass(string ptmListLocation)
```

Comment:
Sounds good! I got that syntax from MetaMorpheus [here](https://github.com/smith-chem-wisc/MetaMorpheus/blob/master/TaskLayer/CalibrationTask/CalibrationTask.cs#L93), but agreed that OfType is cleaner. I'll update this code.